contactmps/templates/layout.html

```diff
@@ -75,45 +24,14 @@
 
         <div><a href="/embed.html">Embed this on your site</a></div>
         {% endblock social_sharing %}
-
-        <footer>
-          {% block attribution_heading %}
-          <h2>Attribution</h2>
-          {% endblock attribution_heading %}
-          {% block attribution %}{% endblock attribution %}
-        </footer>
-
-        {% block js %}
-        {% javascript "pym.js" %}
-        <script>
-          var pymChild = new pym.Child({
-            id: "contactmps-embed-parent"
-          });
-        </script>
-
-        {% javascript "js" %}
-
-        {% if GOOGLE_ANALYTICS_ID %}
-        <script>
-          (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-          (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-          m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-          })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-          ga('create', '{{ GOOGLE_ANALYTICS_ID }}', 'auto');
-          ga('send', 'pageview');
-        </script>
-        {% endif %}
-
-        <script>
-          window.onload = ((function() {
```

Comment:
I think it's better to stick with window.onload for this - the idea is that onload happens after assets have _loaded_, and we explicitly want to ensure the height is right. There's no reason to send the height on $() == $(document).ready which is just when we can start editing the dom
https://stackoverflow.com/a/5182050
I think in theory this shouldn't be necessary, but since it's been useful to be really sure the window is the right height, better to use onload than ready.